### PR TITLE
Feature/arpa-1

### PR DESCRIPTION
**Pull request by naverus**
Agregar administración de roles de usuarios.

### CODE REVIEW

File: app/arpa-app.html [hunk 1]
```diff
@@ -171,6 +180,25 @@
             _setupUserSettings: function (_user) {
```
Comment (nescaler) [inline app/arpa-app.html]: add documentation and comments required.

File: app/arpa_components/arpa-app-elements/arpa-common-behaviors.html [hunk 2]
```diff
@@ -103,6 +103,22 @@
                 if (this.loadResources) {
                     this.loadResources(this.resolveUrl('locales.json'));
                 }
+            },
+
+            isRoot: function(_userRole) {
```
Comment (nescaler) [inline app/arpa_components/arpa-app-elements/arpa-common-behaviors.html]: add documentation and comments

File: app/arpa_components/arpa-user-roles/arpa-user-roles.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                            userRole.localizedRole = this.localize(userRole.role);
+                        }
+                        this.set("userRoles", userRolesTemp);
+                    }.bind(this));
+                }
+            },
+
+
+            /**
+             * Handles save action to update current roles that is being displayed.
```
Comment (nescaler) [inline app/arpa_components/arpa-user-roles/arpa-user-roles.html]: complete the documentation

File: app/arpa-app.html (hunk 1 above)
Comment (naverus) [reply app/arpa-app.html]: Fixed.

File: app/arpa_components/arpa-app-elements/arpa-common-behaviors.html (hunk 2 above)
Comment (naverus) [reply app/arpa_components/arpa-app-elements/arpa-common-behaviors.html]: Fixed.

File: app/arpa_components/arpa-user-roles/arpa-user-roles.html (hunk 3 above)
Comment (naverus) [reply app/arpa_components/arpa-user-roles/arpa-user-roles.html]: Fixed.

**Pull request merged**